company_values.md

```diff
@@ -0,0 +1,15 @@
+## Company values
+
+Our company follow a culture based on three pillars:
+
+ - _Discipline_: Be constant and perseverant to achieve the goals.
+ - _Pragmatism_: The result matter.
+ - _Meritocracy_: All effort is awarded.
+
+
+## Member values
+
+We charge the members of our company to govern their lives based on principles. 
+They are rules and immutable values, e.g. Honesty in china is same as honesty in Perú.
```

Comment:
Change the following part:
They are rules and inmutable values -> They are immutable rules and values.